File: v2/csharp/src/main/csharp/vestorly/Model/NewsletterSettings.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class NewsletterSettings {
    
    
    [DataMember(Name="newsletter_settings", EmitDefaultValue=false)]
    public List<NewsletterSetting> newsletterSettings { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class NewsletterSettings {\n");
      
      sb.Append("  newsletterSettings: ").Append(newsletterSettings).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/NewsletterSettings.java
package vestorly.client.model;

import vestorly.client.model.NewsletterSetting;
import java.util.*;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class NewsletterSettings  {
  
  private List<NewsletterSetting> newsletterSettings = new ArrayList<NewsletterSetting>() ;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("newsletter_settings")
  public List<NewsletterSetting> getNewsletterSettings() {
    return newsletterSettings;
  }
  public void setNewsletterSettings(List<NewsletterSetting> newsletterSettings) {
    this.newsletterSettings = newsletterSettings;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class NewsletterSettings {\n");
    
    sb.append("  newsletterSettings: ").append(newsletterSettings).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
